rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSet.java

```diff
@@ -680,6 +682,23 @@
                                                                           .commandClass(AddNodeCommand.class)
                                                                           .entry();
 
+    public static final CommandSet.Entry NS_DEFINE = CommandSetEntryBuilder.newInstance()
+                                                                           .opt("dfns")
+                                                                           .longOpt("definens")
+                                                                           .description("Define new node source")
+                                                                           .hasArgs(true)
+                                                                           .numOfArgs(1)
+                                                                           .hasOptionalArg(true)
+                                                                           .argNames("node-source [nodes-recoverable]")
+                                                                           .jsCommand("definens(node-source[,nodes-recoverable])")
+                                                                           .commandClass(DefineNodeSourceCommand.class)
+                                                                           .entry();
+
+    /**
+     * @deprecated  As of release 7.37, replaced by
+     * {@link CommandSet#NS_DEFINE} and {@link CommandSet#NS_DEPLOY}
+     */
+    @Deprecated
```

Comment:
Is it too early to remove this for the V8?

Reply:
Good catch ;) I didn't plan we were changing major! I will update this everywhere I put it, into 'As of version 8,'

Reply:
Yes, we cannot remove this now, some clients are using scripts that deploy nodes through this command.